definitions/ext-pixie_postgres/summary_metrics.yml

```diff
@@ -0,0 +1,12 @@
+throughput:
```

Comment:
dito for summary metrics https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-postgresqlinstance/summary_metrics.yml